Tests/govuk_ios/govuk_ios_snapshot_tests/Specs/ViewControllers/SearchViewControllerSnapshotTests.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import XCTest
+import UIKit
+
+@testable import govuk_ios
+
+class SearchViewControllerSnapshotTests: SnapshotTestCase {
+    func test_loadInNavigationController_light_rendersCorrectly() {
```

Comment:
Both function bodies here seem identical except for .light and .dark?
To reduce duplication and half the line count here we could just create private func like `getNavigationController() -> UINavigationController` with lines 9-16 in it's body and then set .light and .dark on the returned UINavigationController?

Reply:
Good suggestion. I've created a helper method which is shared across the snapshot specs which DRY things up, see the last commit.